pandas/core/internals/array_manager.py

```diff
@@ -385,7 +385,10 @@ def apply(
 
         return type(self)(result_arrays, new_axes)
 
-    def apply_with_block(self: T, f, align_keys=None, **kwargs) -> T:
+    def apply_with_block(self: T, f, align_keys=None, swap_axis=True, **kwargs) -> T:
+        # switch axis to follow BlockManager logic
```

Comment:
is this needed now that you have a _normalize_axis?

Reply:
Yes, it is still needed here because this is the function that does the "use block method as fallback" (and the block method is still using its inverted axis logic). 

(but I am actively removing the usage of `apply_with_block`, like https://github.com/pandas-dev/pandas/pull/40196)